lib/src/helpers/help_widget_view.dart

```diff
@@ -0,0 +1,17 @@
+/// It's defining a new type called [HelpWidgetView] with three possible values:
+/// `collapsed`, `main`, and `options`.
```

Comment:
It is an enum by definition -- we don't need to state the obvious here. Besides -- it's probably not a good idea to list all the values in a doc comment -- it easily get out of date and confuses the reader.

```suggestion
/// A listing of available views of a single HelpUkraineWidget.
```